dev/com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties

```
@@ -30,9 +30,24 @@ max=Maximum concurrency
 max.desc=Specifies the maximum number of tasks that can run simultaneously. The default is Integer.MAX_VALUE. Maximum concurrency can be updated while tasks are in progress. If the maximum concurrency is reduced below the number of concurrently running tasks, the update goes into effect gradually as in-progress tasks complete, rather than canceling them.
 
 maxPolicy=Maximum policy
-maxPolicy.desc=Indicates whether to loosely or strictly enforce maximum concurrency for tasks that run on the task submitter's thread. Tasks can run on the task submitter's thread when using the untimed invokeAll method, or, if only invoking a single task, the untimed invokeAny method. If the run-if-queue-full attribute is configured, it is also possible for tasks to run the task submitter's thread when using the execute and submit methods. In all of these cases, this attribute determines whether or not running on the submitter's thread counts against the maximum concurrency.
-maxPolicy.loose.desc=Maximum concurrency is loosely enforced. Tasks are allowed to run on the task submitter's thread without counting against maximum concurrency.
-maxPolicy.strict.desc=Maximum concurrency is strictly enforced. Tasks that run on the task submitter's thread count towards maximum concurrency. This policy does not allow tasks to run on the task submitter's thread when already at maximum concurrency.
+maxPolicy.desc=Indicates whether to loosely or strictly enforce maximum \
+ concurrency for tasks that run on the task submitter's thread. If the \
+ concurrency policy is not configured to run tasks on virtual threads, tasks can \
+ run on the task submitter's thread when using the untimed invokeAll method, or, \
+ if only invoking a single task, when using the untimed invokeAny method. \
+ If the run-if-queue-full attribute is configured, it is also possible for tasks \
+ to run the task submitter's thread when using the execute and submit methods. \
```

```suggestion
 run on the task submitter's thread when using the execute and submit methods. \
```

here also- avoid "when using...", instead make clear who/what is using